container/Dockerfile

```diff
@@ -1,21 +1,23 @@
 # Stage 1: Build RPMs
 
-FROM fedora:32
+FROM fedora:36
 USER root
+RUN dnf --assumeyes install 'dnf-command(copr)' && dnf --assumeyes copr enable mgrabovs/python-wtforms-sqlalchemy
```

Comment:
I think I'd prefer to move the repo under the abrt Copr org/group.

Reply:
Actually, I already have a [package review request](https://bugzilla.redhat.com/show_bug.cgi?id=2055622) open that I forgot about.

Reply:
I'm actually going to build the package in the [faf-el8 Copr](https://copr.fedorainfracloud.org/coprs/g/abrt/faf-el8/builds/) so that we can pull it here.

Reply:
@msrb It's built now, can you please update the PR?

Reply:
Hm, it just occurred to me that we might need to build a couple more packages for all this to work correctly on RHEL 8, but I don't have the capacity to look into it right now.

Reply:
@mgrabovsky sorry for the delay here.

I will try to update things here by the end of this week.